app/dance/movers/momentum.go

```diff
@@ -66,35 +91,80 @@ func (bm *MomentumMover) SetObjects(objs []objects.BaseObject) int {
 		}
 	}
 
+	s, ok1 := end.(*objects.Slider)
+	if ok1 { ok1 = !s.IsRetarded() }
+
+	// stream detection logic stolen from spline mover
+	stream := false
+	ms := settings.Dance.Momentum
```

Comment:
If a variable is used later (not only in the scope of if block) it's preferable to not stick it here. You can move it above the comment (ofc leaving one blank line before and after)